packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -25,9 +32,46 @@ export default defineComponent<KottiButton.PropsInternal>({
 	name: 'KtButton',
 	props: makeProps(KottiButton.propsSchema),
 	setup(props, { emit, slots }) {
+		const contentRef = ref<Element | null>(null)
+		const triggerRef = ref<Element | null>(null)
+
 		const hasSlot = computed(() => Boolean(slots.default))
```

Comment:
@Juli0GT i just realised that a button can accept a slot (which can accept text or icons or anything) and I'm not sure what the support is like for those buttons

because they can have a label through the slot, not through the label prop